lib/enkf/analysis_config.cpp

```diff
@@ -430,7 +430,7 @@ void analysis_config_init( analysis_config_type * analysis , const config_conten
     analysis_config_set_std_cutoff( analysis , config_content_get_value_as_double( config , STD_CUTOFF_KEY ));
 
   if (config_content_has_item( config , ENKF_ALPHA_KEY ))
-    analysis_config_set_alpha( analysis , config_content_get_value_as_double( config , ENKF_ALPHA_KEY ));
+    fprintf(stdout, "Setting ENKF_ALPHA directly is deprecated and should be changed to UPDATE_SETTING ENKF_ALPHA to be taken into consideration!");
```

Comment:
Is it possible to use `config_parser_deprecate` as it is done [here](https://github.com/equinor/libres/blob/83f59a78471591ff105118ddd7823e53554f0c68/lib/enkf/analysis_config.cpp#L692)?

Reply:
will have a look! thx 👍